.github/workflows/run_tests.yml

```diff
@@ -37,34 +37,29 @@ jobs:
           # - 2.11.1
           - latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install ".[dev]"
     - name: Lint with flake8
       run: |
-        pip install flake8
-        pip install pandocfilters
         # stop the build if there are Python syntax errors or undefined names
         flake8 ./panflute --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 ./panflute --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Download Pandoc
       run: |
-        pip install requests
-        downloadUrl=$(python -c 'import requests, sys; v = sys.argv[1]; url = "https://api.github.com/repos/jgm/pandoc/releases/" + ("" if v == "latest" else "tags/") + v; print(next(i["browser_download_url"] for i in requests.get(url).json()["assets"] if i["name"][-3:] == "deb"))' ${{ matrix.pandoc-version }})
-        wget $downloadUrl
+        downloadUrl=$(python .github/workflows/pandoc_url.py ${{ matrix.pandoc-version }})
```

Comment:
For this and the pandoc_url.py, why don't we hold this off first? It is likely I'm reverting to that without GitHub API.